mllib/src/main/scala/org/apache/spark/mllib/evaluation/RegressionMetrics.scala

```diff
@@ -31,7 +31,8 @@ import org.apache.spark.sql.DataFrame
  * @param predictionAndObservations an RDD of (prediction, observation) pairs.
  */
 @Experimental
-class RegressionMetrics(predictionAndObservations: RDD[(Double, Double)]) extends Logging {
+class RegressionMetrics(@transient predictionAndObservations: RDD[(Double, Double)])
+  extends Logging with Serializable {
```

Comment:
I see, I misunderstood yall's comments.  Does that mean you couldn't get it to work just by marking the summary field in the model transient?